assets/_scss/core/_defaults.scss

```diff
@@ -0,0 +1,52 @@
+// Starter Kit Defaults
+// Do not modify.
+// These variables can be modified in _variables.scss
+
+// Typography
+$em-base:       10px; // default is already defined in bourbon
+$base-font-size:    rem(16px) !default;
+$small-font-size:   rem(14px) !default;
+$title-font-size:   rem(48px) !default;
+$h1-font-size:      rem(36px) !default;
+$h2-font-size:      rem(28px) !default;
+$h3-font-size:      rem(24px) !default;
+$h4-font-size:      rem(18px) !default;
+$h5-font-size:      rem(16px) !default;
+$h6-font-size:      rem(14px) !default;
+$base-line-height:    1.5em !default;
+$heading-line-height: 1.3em !default;
+
+$font-sans:       'Source Sans Pro', 'Helvetica', 'Arial', sans-serif !default;
+$font-serif:      'Georgia', 'Times New Roman', serif;
+$font-mono:       'Source Code Pro', 'Courier New', 'Courier', monospace !default;
+
+
+// Color
+$color-grey-50:     #fafafa !default;
+$color-grey-100:    #f5f5f5 !default;
+$color-grey-200:    #eeeeee !default;
+$color-grey-300:    #e0e0e0 !default;
+$color-grey-400:    #bdbdbd !default;
+$color-grey-500:    #9e9e9e !default;
+$color-grey-600:    #757575 !default;
+$color-grey-700:    #616161 !default;
+$color-grey-800:    #424242 !default;
+$color-grey-900:    #212121 !default;
+
+$color-base-link:   #000000 !default;
+
+$color-main:        #0071bc !default;
+$color-main-dark:   #00609f !default;
+
+// Mobile First Breakpoints
+$mobile-up:       481px !default;
+$tablet-up:       600px !default;
+$widescreen-up:     1201px !default;
+
+
+// Magic Numbers
+$site-max-width:    960px !default;
+$site-margins:      40px !default;
+$article-max-width:   600px !default;
+$border-radius:     0px !default;
+$box-shadow:      0 0 2px rgba(#000, .3) !default;
```

Comment:
Nitpicky fix. Please align the values on rows 42 - 52 (similarly to how you did the color values.